create-manifest.js

```diff
@@ -1,12 +1,14 @@
-
-const fs = require('fs');
+const fs = require('fs-extra')
 const path = require('path');
+const jsonFormat = require('json-format');
 
 const createManifest = function (payload) {
-  fs.mkdir(path.join(__dirname, payload.name), function(err, dir) {
-    const manifestStream = fs.createWriteStream(path.join(__dirname, payload.name, 'manifest.json'));
-    manifestStream.write(JSON.stringify(payload));
-  })
+	fs.ensureDir(path.join(__dirname, payload.name))
+		.then((dir) => {
+			fs.outputFile(`${dir}/manifes.json`, jsonFormat(payload))
+				.catch(err => { throw err });
```

Comment:
You can write it as `return fs.outputFile` instead and remove catch so that final catch will be able to catch all the intermediate errors.